packages/i18n-translate/src/index.js

```diff
@@ -0,0 +1,82 @@
+import _ from 'lodash';
+import { sprintf } from 'sprintf-js';
+
+const pluralizeFunctions = {
+  en: number => (number === 0 || number > 1 ? 'other' : 'one'),
+  fr: number => (number > 1 ? 'other' : 'one'),
+  hu: (number, general) => {
+    if (!general) {
+      return 'one';
+    }
+
+    return number > 1 ? 'other' : 'one';
+  },
+  hr: (number, general) => {
+    // General plural
+    if (general) {
+      return number > 1 ? 'other' : 'one';
+    }
+
+    const numberInString = number.toString();
+    const lastDigit = numberInString.charAt(numberInString.length - 1);
+
+    if ((number > 4 && number < 21) || ['0', '5', '6', '7', '8', '9'].includes(lastDigit)) {
+      // Third plural form
+      return 'many';
+    } else if (lastDigit === '1') {
+      // First plural form and singular
+      return 'one';
+    } else if (lastDigit === '2' || lastDigit === '3' || lastDigit === '4') {
+      // Second plural form
+      return 'few';
+    }
+
+    return '';
+  },
+};
+
+export const translate = (lang, i18nNames = {}, errorCallback = _.noop, renderersHandler = _.noop) => {
+  const pluralize = pluralizeFunctions[_.get(lang, '_i18n.lang')] || pluralizeFunctions.fr;
+
+  return (key, data = {}, number, general, renderers) => {
+    let combineKey = key;
+    // Pluralize
+    if (typeof number !== 'undefined') {
+      combineKey = `${key}.${pluralize(number, general)}`;
+    }
+
+    let translation;
+    if (_.has(lang, combineKey)) {
```

Comment:
```suggestion
    if (_isString(lang[combineKey]))
```
There may be a case when we mistakenly accessing wrong nesting level and receiving an object instead of string. For example, given such a translations data:
```
const lang = { "a" : { "b": { "c": { "d": "actual value" } } } }
```
the `_.has(lang, 'a.b.c')` condition will be true. Then the selected object `{"d": "actual value"}` will be passed to `sprintf` which will throw an error here: https://github.com/alexei/sprintf.js/blob/00af1e6fe67bdbd6928e03537cd14aa27752b936/src/sprintf.js#L204